docs/content/configuration/storage.md

```diff
@@ -48,6 +48,27 @@ Disk storage allows modules to be stored on a file system. The location on disk
 
 where `/path/on/disk` is your desired location. Also it can be set using `ATHENS_DISK_STORAGE_ROOT` env
 
+## HTTP
+
+This driver stores files to an HTTP server via standard GET and PUT requests. The files are laid out in a manner identical to the proxy URL used to access them, and the requests are optionally (but hopefully!) authenticated using Basic Auth.
+
+The HTTP storage driver can be used to integrate with systems like Artifactory that offer blob storage over HTTP.
```

Comment:
nit: can you mention that this is different from other blob storage systems like S3, Azure blob storage, or Google Cloud Storage that also implement (proprietary) HTTP interfaces?